xcp_d/utils/utils.py

```diff
@@ -313,10 +313,22 @@ def butter_bandpass(
     """
     from scipy.signal import butter, filtfilt
 
+    if low_pass == 0 and high_pass > 0:
+        btype = "highpass"
+        filt_input = high_pass
+    elif low_pass > 0 and high_pass == 0:
+        btype = "lowpass"
+        filt_input = low_pass
+    elif low_pass > 0 and high_pass > 0:
+        btype = "bandpass"
+        filt_input = [high_pass, low_pass]
```

Comment:
I think this could be simplified if you put bandpass at the top. E.g., `if low_pass > 0 and high_pass > 0... elif low_pass > 0... elif high_pass > 0`. I don't think we need to worry about negative values (i.e., they should just be treated as 0s).

Reply:
done

Reply:
thx!